src/main/java/com/sourceclear/marvin/Marvin.java

```
@@ -122,6 +131,31 @@ public Model fetchMavenPomFile(String groupId, String artifactId, String version
 
 
   /**
+   * Gets the license.txt file if it exists
+   * @param groupId the group id
+   * @param artifactId the artifact id
+   * @param version  the version
+   * @return an input stream of the file
+   */
+  public InputStream fetchLicenseTxt(String groupId, String artifactId, String version) {
+    try {
+      String url = createMavenLicenseFileFromId(groupId, artifactId, version);
+      HttpResponse<InputStream> licenseFile = Unirest.get(url).asBinary();
+
+      if (licenseFile.getStatus() != HttpStatus.SC_OK) {
+        return null;
+      }
+
+      return licenseFile.getBody();
+    } catch ( UnirestException ex) {
+      LOGGER.error("Couldn't get the licesne file for coordinates.", ex);
+    }
+    return null;
```

per above comment, maybe better to rethrow as an IOException instead of returning null. Seems like clients should be able to use exception handling in this case instead of null checks.